src/domain/member.ts

```diff
@@ -64,6 +64,8 @@ export function updateMemberTokens(
   let dao = getDAO(daoAddress.toHex());
   let token = DAOToken.bind(hexToAddress(dao.nativeToken));
   let member = getMember(address, daoAddress);
-  member.tokens = token.balanceOf(address);
-  saveMember(member);
+  if (member.reputation.toString().charAt(0) !== '0') {
```

Comment:
is there a  case where updateMemberToken is called where the rep is 0?

Reply:
Yes, I think this function is called on every token transfer so it might not actually be between members. That's more of a bugfix.